### ISSUE TITLE
OpenAPI Certification

### ISSUE THREAD

**Issue by kinlane** [kinlane/oai#10]
Augmenting the "compliance stamp" idea above, let's widen certification of people, services, and open tooling. Augment the education program with it. Work with other providers in the space to support an OpenBadges certification program with OAI as the issuer.

**Comment by kevinswiber** [comment]
I think training programs, run by OAI members and/or OpenAPI tooling vendors and blessed by the OpenAPI Initiative might be a good starting point.  Vendors have a benefit to provide a certification program.  OAI has a benefit in ensuring the programs are covering the bases.  I imagine there are a few folks who might be interested in this today.

**Comment by mniinio** [comment]
I agree with Kevin, and interested to work on this more. I'd suggest starting from key pre-requisite skills + key work-related goals that people should be able to accomplish after being certified. It should be quite generic, but related to the specific main version of OpenAPI (or a yearly update of requirements) since there are big differences with specific versions on the detail level, but not on the "overview" level

**Comment by jglassenberg** [comment]
I'd be happy to support this.  It may be easy to create a certification exam, and follow on with supportive training around the exam.

**Comment by mniinio** [comment]
I propose we divide this into tasks, something like this:

1. Decide the target audience for the first exam (total newbies to OpenAPI, do we assume JSON or YAML skills and REST / AsyncAPI understanding? Developers who'd generate it from code or architects etc who would handcraft it.)
2. Outcomes: What it is that they should be able to do after completing the exam?
3. Topics: when to use it, where to use it, ways of creating it, general syntax, different parts of the spec
4. Certification training: Is everyone free to organize training, but if you want to give out the certification you need to comply with some rules? Is this going to be a get certified after training (like Certified Scrum Product Owner) or is it going to be a controlled multiple choice etc. type of test

**Comment by MikeRalphson** [comment]
I suspect we should at least coordinate with what the Linux Foundation are already doing in this area. https://training.linuxfoundation.org/certification/

@ncaidin do you have contacts?

**Comment by mniinio** [comment]
@MikeRalphson True, this is what we discussed today at Kin's office hours a bit. But then this means that all training and certifications would be provided by Linux Foundation directly wouldn't it, and would be opposite to @kevinswiber's original comment?

**Comment by ncaidin** [comment]
As an informational point, yes, the Linux Foundation has Training and Certification division. Whichever way the community decides to go, it does seem like a good idea to understand what those options are, imho. Please let me know if/when you wish me to dig further!

**Comment by mniinio** [comment]
@ncaidin I'd say we need you to dig further now since if we don't know the options, we don't know if we want to invest our time or how much we want to do it. Most people in this thread are somehow or another in the consulting and training business

**Comment by kevinswiber** [comment]
@MikeRalphson @mniinio I forgot about LF's training program! I was with NodeSource at the time the Node.js certifications came about. We were tied to OpenJS Foundation, actually helped developed the training material, and scored the tests. I imagine the model might look similar for OAI members.
